setup.py

```diff
@@ -10,7 +10,7 @@ def read(fname):
 
 setup(
     name="netflix-spectator-py",
-    version="0.1.18",
+    version="0.1.19",
```

Comment:
This is a big enough change it should probably go to `0.2.0`

Reply:
That's a good idea. Alternatively, we could consider pushing it to 1.0, since this should be the final form of the library.